schema/schema.yaml

```diff
@@ -589,4 +589,86 @@ tables:
         expression: radial_velocity_error_kms >= 0
 
 
-version: "v0.1"
\ No newline at end of file
+  - name: CompanionRelationships
+    "@id": "#CompanionRelationships"
+    description: Relationships between sources
+    primaryKey: 
+    - "#CompanionRelationships.source"
+    - "#CompanionRelationships.companion" # or companion_name
+
+    columns:
+    - name: source
+      "@id": "#CompanionRelationships.source"
+      datatype: string
+      length: 100
+      description: Main identifier for an object; links to Sources table
+      ivoa:ucd: meta.id;meta.main
+      nullable: false
+    - name: companion
+      "@id": "#CompanionRelationships.companion"
+      datatype: string
+      length: 100
+      description: Identifier for a companion object
+      ivoa:ucd: meta.id
+      nullable: false
+    - name: relationship
+      "@id": "#CompanionRelationships.relationship"
+      datatype: string
+      length: 30
+      description: Relationship of the source to the companion, e.g., "parent", "child", "sibling"
+      nullable: false
+    - name: projected_separation_arcsec
+      "@id": "#CompanionRelationships.projected_separation_arcsec"
+      datatype: double
+      description: Projected separation between the source and companion in arcseconds
+      fits:tunit: arcsec
+      ivoa:ucd: pos.angDistance
+    - name: projected_separation_error_arcsec
+      "@id": "#CompanionRelationships.projected_separation_error_arcsec"
+      datatype: double
+      description: Uncertainty of the projected separation in arcseconds
+      fits:tunit: arcsec
+      ivoa:ucd: stat.error;pos.angDistance
+    - name: comments
+      "@id": "#CompanionRelationships.comments"
+      datatype: string
+      length: 1000
+      description: Free-form comments for this entry
+    - name: reference
+      "@id": "#CompanionRelationships.reference"
+      datatype: string
+      length: 30
+      description: Publication reference; links to Publications table
+      ivoa:ucd: meta.ref
+      nullable: false
+    - name: other_companion_names
+      "@id": "#CompanionRelationships.other_companion_names"
+      datatype: string
+      length: 100
+      description: Additional names for the companion object
```

Comment:
Should we make this field longer or is 100 characters sufficient? Also: we should ask the additional names be in a comma-separated list (or whatever we decide), in case we build tools to parse this.